proxy_agent/src/redirector/windows.rs

```diff
@@ -175,14 +173,14 @@ pub fn lookup_audit(
     bpf_prog::lookup_bpf_audit_map(source_port, shared_state)
 }
 
-pub fn get_audit_from_redirect_context(tcp_stream: &TcpStream) -> std::io::Result<AuditEntry> {
+pub fn get_audit_from_redirect_context(raw_socket: usize) -> std::io::Result<AuditEntry> {
```

Comment:
NIT: I'd rename this to `raw_socket_id` just to make it a bit more explicit, but I'm not hard pressed about it since you can infer that from it's `usize` type